core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/scatter3D/Scatter3dChartOpDesc.scala

```diff
@@ -0,0 +1,122 @@
+package edu.uci.ics.texera.workflow.operators.visualization.scatter3D
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaTitle}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+@JsonSchemaInject(json = """
+{
+  "attributeTypeRules": {
+    "title": "string"
+  }
+}
+""")
+class Scatter3dChartOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+  @JsonProperty(value = "title", required = true)
```

Comment:
Please give a default value for the title by specifying `defaultValue = "XXX"`. in `@JsonProperty`.

Reply:
Fixed.

Reply:
No you have not added the default value for this property yet.